### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by janhavisinghh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 ## Next Release
 
+### Changes
+
+- Ask storage permission when download/share button is clicked
```
Comment (msasikanth) [inline CHANGELOG.md:7]: Can we move this to internal change, please?

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@
 
 ### Internal
 
+- Ask storage permission when download/share button is clicked
+- Add common download function for downloading overdue list
```
Comment (msasikanth) [inline CHANGELOG.md:8]: I don't think this change made it into yesterday's release right? 😅

Comment (janhavisinghh) [reply CHANGELOG.md:8]: Yeah, it didn't. I added this in the next release section

Comment (msasikanth) [reply CHANGELOG.md:8]: Oh right, this is for the main branch. My bad 😅

**Review by msasikanth — APPROVED** [review]

**Pull request merged**